torch/_inductor/kernel/mm.py

```diff
@@ -339,6 +340,15 @@ def tuned_addmm(inp, mat1, mat2, *, alpha=1, beta=1, layout=None):
                 beta=beta,
             )
 
+    if use_ck_template(layout, m, n, k):
```

Comment:
Shall we do 
`if static_shape and is_nonzero and use_ck_template(layout, m, n, k):` ?